google/cloud/compute_v1/types/compute.py

```diff
@@ -6278,14 +6278,14 @@ class Allowed(proto.Message):
             ["12345-12349"].
     """
 
-    i_p_protocol = proto.Field(proto.STRING, number=323774237, optional=True,)
+    I_p_protocol = proto.Field(proto.STRING, number=488094525, optional=True,)
```

Comment:
@vam-google Did the field naming logic change?

Reply:
Yes, it is expected- it is only for this very limited set of fields. For whatever reason, for a few fields in compute API surface, they named the fields (i.e. not messages, fields) as `IPAddress`, not `ipAddress`. When converted to snake_case in proto, if we convert that as `i_p_address`, or `ip_address`, when serialized to JSON they become `iPAddress`, `ipAddress` and  respectively, which is not what server expects (it expects `IPAddress`).